packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -78,6 +78,12 @@ export const getIsScreenSharingOn = (state: CallClientState, props: CallingBaseS
 export const getIsMuted = (state: CallClientState, props: CallingBaseSelectorProps): boolean | undefined =>
   state.calls[props.callId]?.isMuted;
 
+/**
+ * @private
+ */
+export const getOptimalVideoCount = (state: CallClientState, props: CallingBaseSelectorProps): number | undefined =>
+  state.calls[props.callId]?.optimalVideoCountFeature.optimalVideoCount;
```

Comment:
Is this a beta feature currently or in stable builds of calling sdk? If its beta we'll need to add conditional compile flags [(adding beta only features)](https://github.com/Azure/communication-ui-library/blob/main/docs/references/beta-only-features.md#adding-features-to-be-released-in-the-beta-channel-only)

Reply:
Addressed this comment by adding conditional compile flags